pkg/appmanager/resourceConfig.go

```diff
@@ -1512,10 +1543,6 @@ func (appMgr *Manager) createRSConfigFromIngress(
 	}
 
 	log.Debugf("%d whitelist sources", len(whitelistSourceRangeRefs))
-	if len(urlRewriteRefs) > 0 || len(appRootRefs) > 0 || len(whitelistSourceRangeRefs) > 0 {
-		cfg.MergeRules(appMgr.mergedRulesMap)
-	}
```

Comment:
Does this need to be removed? Will this cause us to miss the whitelistSourceRangeRefs case?

Reply:
I think this is fine, but I will talk to Tim Rupp about it, since merging rules only really matters when we have the url-rewrite or app-root annotations and this will be called at the end syncRoutes if those annotations are applied.